lib/resources/dstu2/soarian_diagnostic_report.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: DiagnosticReport
+field_name_base_url: http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport
+fields:
+
+- name: code
+  required: 'No'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: A code or name that describes this diagnostic report.
+  binding:
+    description: Codes that describe Diagnostic Reports.
+    terminology:
+    - display: LOINC Diagnostic Report Codes
+      system: http://hl7.org/fhir/ValueSet/report-codes
```

Comment:
This should link to the DSTU 2 documentation: http://hl7.org/fhir/dstu2/valueset-report-codes.html